packages/dds/matrix/src/matrix.ts

```diff
@@ -230,32 +230,36 @@ export class SharedMatrix<T extends Serializable = Serializable>
 
         this.cells.setCell(rowHandle, colHandle, value);
 
-        this.sendSetCellOp(row, col, value, rowHandle, colHandle);
-    }
-
-    private sendSetCellOp(row: number, col: number, value: T, rowHandle: Handle, colHandle: Handle) {
-        // If the SharedMatrix is local, it will by synchronized via a Snapshot when initially connected.
-        // Do not queue a message or track the pending op, as there will never be an ACK, etc.
         if (this.isAttached()) {
-            const localSeq = this.nextLocalSeq();
+            this.sendSetCellOp(row, col, value, rowHandle, colHandle);
+        }
+    }
```

Comment:
Ugh.  What an awful diff for a simple change:

1.  The 'isAttached()' check is hoisted from 'sendSetCellOp()' to the caller.  (above ---^)
2.  'sendSetCellOp()' now takes 'localSeq' as an argument.   (below ----v)
3.  ...which forced me to reformat the signature due to line length (below ---v)

The motivation for #1 is to pedantically avoid advancing 'nextLocalSeq' when no op will be sent.
The motivation for #2 is to enable 'reSubmitCore' to preserve the 'localSeq' from the original op.